src/html/rangeTemplate.html

```diff
@@ -12,11 +12,11 @@
         </div>
         <div class="small-7 medium-6 end columns">
             <div class="row">
-                <input type="button" value="+" class="gpiic-fd-range-increase left gpii-fd-increase" id="gpiic-fd-increase" tabindex="1" />
+                <button class="gpiic-fd-range-increase left gpii-fd-increase" id="gpiic-fd-increase"><span class="gpii-fd-icon"></span></button>
```

Comment:
Is using a font icon necessary in this case? Would using plain text make more sense?